src/systems/ec/implementations/react/components/date-block/src/DateBlock.jsx

```diff
@@ -16,26 +16,26 @@ const DateBlock = ({
 
   return (
     <div {...props} className={classNames}>
-      <div className="ecl-date-block__week-day">{weekDay}</div>
       <div className="ecl-date-block__day">{day}</div>
       <div className="ecl-date-block__month">{month}</div>
```

Comment:
it could be nice to use a `<abbr>` tag with `title` attribute set to the month's full name